build.gradle

```diff
@@ -593,19 +517,6 @@ dependencyCheck {
     suppressionFile = 'config/owasp/suppressions.xml'
 }
 
-dependencyUpdates.resolutionStrategy = {
-    componentSelection { rules ->
-        rules.all { ComponentSelection selection ->
-            boolean rejected = ['alpha', 'beta', 'rc', 'cr', 'm'].any { qualifier ->
-                selection.candidate.version ==~ /(?i).*[.-]${qualifier}[.\d-]*/
-            }
-            if (rejected) {
-                selection.reject('Release candidate')
-            }
-        }
-    }
-}
-
```

Comment:
resolution strategy ensures only released versions are being used. why is it being removed?

Reply:
Good spot, my mistake. This should've stayed.